README.md

```diff
@@ -55,6 +56,36 @@ The following settings are supported in [CocConfig][9]:
 
 - `java.debug.vimspector.profile` : **(Deprecated)** Set to `null` and use `"default":true` in Vimspector.json instead. Specifies the Vimspector [profile][10] to activate when launching. Set to `null` to be prompted if multiple configurations are found and no default is set. Defaults to `Java Attach`
 - `java.debug.vimspector.substitution.adapterPort` : Specifies the Vimspector [adapter port][11] substitution name in `.vimspector.json`. The actual port number will replace this value in the Vimspector config when the debug server is started. Defaults to `AdapterPort`
+- `java.debug.settings.showHex`: show numbers in hex format in "Variables" viewlet, defaults to `false`.
```

Comment:
Is there a reason `java.debug.logLevel` documentation is not included here?

Reply:
After looking into this more, I think it should match the vscode-java-debug configuration and use `java.debug.logLevel` to avoid confusion and it should be documented here.

Reply:
The reason why I didn't include `logLevel` doc is that I didn't find the way how it can be "used" in our case. I don't see any changes in neither vimspector nor language server logs when changing this level.
I suspect it is something VS Code specific.
Do you have an idea where we can observe these logs?

Reply:
aha, indeed, these logs goes to language server logs.
With `verbose` level there is much more logs there.
I will add it doc, config and will use the same property as in VS code (as proposed)

Reply:
Thanks!